ffcli/command_test.go

```diff
@@ -439,6 +439,65 @@ func TestIssue57(t *testing.T) {
 	}
 }
 
+func TestSharedFlags(t *testing.T) {
+	for _, testcase := range []struct {
+		name       string
+		args       []string
+		sharedFlag string
+	}{
+		{
+			name:       "foo",
+			args:       []string{"foo"},
+			sharedFlag: "foo-default",
```

Comment:
This testcase fails, when printing usage, we can correctly see `foo-default`, but when you run the foo command without CLI flags, then you get `bar-default`.

I tried to add some printf-debugging in the library and I can't find why the pointer is modified because, the `ff.Parse` function is only called against the root's and foo's `ffcli.Command.FlagSet`s

Any idea what's the root cause and how to make a workaround?